### ISSUE TITLE
export models in ATDB format

### ISSUE THREAD

**Issue by hunterowens** [CityOfLosAngeles/count-machine#14]
[SCAG](http://www.scag.ca.gov/programs/Pages/ActiveTransportationFunding.aspx) has an Active Transportation Program, which maintains a 6 county wide active transportation count database. 

We should have the counts export in a `sqlite` db / CSV compatable with this format. Attaching the schema. 

[ATDB database schema_version22.pdf](https://github.com/CityOfLosAngeles/count-machine/files/3764047/ATDB.database.schema_version22.pdf)

[ATDB table script.txt](https://github.com/CityOfLosAngeles/count-machine/files/3764048/ATDB.table.script.txt)

Contact 
"Kimberly S. Clark" <Clark@scag.ca.gov>,
Hannah Keyes <keyes@scag.ca.gov> with questions about the schema.

**Comment by ian-r-rose** [comment]
@hunterowens Is this format documented anywhere, and has it been adopted more widely? I am all about following specs and standards, but I am a bit worried that this one is (a) very complex, and (b) abandoned.

The link I found to a "Bicycle Count Data Clearinghouse" appears to be broken: http://www.scag.ca.gov/programs/Pages/ActiveTransportation.aspx

**Comment by hunterowens** [comment]
My understanding is SCAG is continuing to work on the project, but we should probably have a call with them to get an update. 

The judgement that we should make, IMO, is if there really is a regional ped / bike count database that _only_ accepts data in ^ format, we should partake in the efforts, otherwise, we should just report in a lighter weight format. (csvs on the open data portal). 

I am also curious if there are other efforts a la MDS or GTFS-Ride for this type of data.

**Comment by hunterowens** [comment]
update: call was had, got access to standard counter API docs. I probably shoulda updated this tracking issue at the time, but will do my best to take a cut at a later date.

**Comment by hunterowens** [comment]
did we update the docs on this / do we want to make a JSON Schema or spec for this?

**Comment by ian-r-rose** [comment]
No, this is still a TODO. We'd like to be able to define an abstract interface for outputs so that we can write different output formats with the flip of a switch (CSV, ATSAC loop counter, ATDB), but that has yet to be done.
